### ISSUE TITLE
CloseButton: No reader text

### ISSUE THREAD

**Issue by EldRoos** [seb-oss/green#949]
### Bug already reported ?

- [X] I confirm that I have checked if the bug already has been reported

### Component name

Close Button

### Description

The close button does not have any name so reader just says button - check for example AlertRibbon, Card, Modal/Dialoge. 

Might be connected to this issue: #1139

### Expected Behaviour

Maybe use title="close", but should be in correct language.
https://accessibilityinsights.io/info-examples/web/button-name/

**Comment by EldRoos** [comment]
Looks like this could be fixed now when I'm testing.

**Comment by EldRoos** [comment]
Still bad.
In Chlorophyll: Says **button** on Tab and nothing on Hover.
In Angular: Says **button Close** on Tab and nothing on Hover.
In React: Says **button** Tab and nothing on Hover.

**Comment by EldRoos** [comment]
Fixed
